phpstan.neon

```diff
@@ -1,7 +1,29 @@
+includes:
+	- phpstan-baseline.neon
+
 parameters:
-    level: 1
+    level: max
+
+    checkFunctionNameCase: true
+    checkInternalClassCaseSensitivity: true
+    treatPhpDocTypesAsCertain: false
+
     paths:
         - application
         - library
+
     scanDirectories:
         - vendor
+
+    ignoreErrors:
+        -
+            messages:
+                - '#Unsafe usage of new static\(\)#'
+                - '#. but return statement is missing#'
+            reportUnmatched: false
+
+    universalObjectCratesClasses: # to ignore magic property errors
+        - ipl\Orm\Model
+        - Icinga\Web\View
+        - Icinga\Data\ConfigObject
+        - Zend_Form_Element
```

Comment:
Please add it to the baseline file instead.

Reply:
Is this even required since Web uses the new Zend fork?

Reply:
I mean, the `Zend_Form_Element`. The other's are fine as they don't resemble issues that will be fixed, rather than stuff we want to ignore. In the base file should be only stuff we want to fix.

Reply:
> Is this even required since Web uses the new Zend fork?

Removed!